common/CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ add_library(common
         util/os.cpp
         util/print_float.cpp
         util/FontUtils.cpp
-        util/FrameLimiter.cpp)
```

Comment:
make sure to fix this

Reply:
Are you referring to how the quoted style slipped in on one of my earlier PRs? https://github.com/open-goal/jak-project/blob/master/common/CMakeLists.txt#L51 there is no change to this file in this PR's diff.

Reply:
oops? I guess might as well fix it now.

Reply:
I can, just wasn't sure what you were referring to